hu.irl.keystep37-settings.yml

```diff
@@ -0,0 +1,33 @@
+app-id: hu.irl.keystep37-settings
```

Comment:
can you put something up in https://irl.hu/ so that I can tell you control the domain ?

A link to github repo will do.

Reply:
I've added it